README.md

```diff
@@ -34,8 +33,8 @@ Read more about `libvcx` deprecation and `unifii` benefits [here](https://github
 - We welcome new contributors! Connect with us via the channels above and take a look at [CONTRIBUTING.md](CONTRIBUTING.md)
 
 ## Versioning & releases
-- We are currently not following semantic versioning. Version are releasing `0.x.x` versions. 
+- We are currently not strictly following semantic versioning and all releases are `0.x.x`. 
```

Comment:
I would suggest rather something like "all releases have major version 0 (i.e. the full version follows the pattern 0.x.y)".